flows/yosys/build_defs.bzl

```diff
@@ -0,0 +1,97 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Reimplementing place-and-route using composable and externalizable pieces"""
+
+load("//flows:flows.bzl", "FlowStepInfo", "script_prefix")
+load("//pdk:build_defs.bzl", "StandardCellInfo")
+
+def _yosys_synth_file_step_impl(ctx):
+    yosys_executable = ctx.attr._yosys.files_to_run.executable
+    yosys_runfiles = ctx.attr._yosys[DefaultInfo].default_runfiles
+
+    yosys_wrapper = ctx.actions.declare_file(ctx.attr.name)
+
+    standard_cells = ctx.attr.standard_cells[StandardCellInfo]
+
+    liberty = standard_cells.default_corner.liberty
+
+    synth_tcl = ctx.file.synth_tcl
+
+    cell_runfiles = ctx.runfiles(files = [synth_tcl, liberty, yosys_wrapper])
+
+    yosys_args = [
+        # "-q", # quiet mode only errors printed to stderr
+        # "-q", # second q don't print warnings
```

Comment:
Do you really want all this info to be printed in shell? I don't feel like that should be the default. @cdleary had previously filed a bug about that.

Reply:
I went back and forth on that because I (mistakenly) thought there wasn't an easy way to undo the quiet flag once it is baked into the wrapper script. While Yosys's -v <level> isn't quite the inverse -v is with other tools, I think it's good enough for people to turn up logging when running the wrapped tool (without having to change things inside the wrapper).

Anyway, I've made the change to quieter by default.